app/mentions/mentions.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var express = require('express');
+//var router  = require('router');
+//var path    = require('path');
+var mongo     = require('../../lib/mongo/');
+var ObjectID = require('mongodb').ObjectID;
+var nodemailer = require('nodemailer');
+var sinon = require('sinon');
+
+var transporter = nodemailer.createTransport({
+  service: 'Gmail',
+  auth: {
+    user: 'test.nodemailer@gmail.com',
+    pass: '123'
+  }
+});
+
+var mailOptions = {
+  from    :'test.nodemailer@gmail.com',
+  to      :'NodemailerTEST@gmail.com',
+  subject : 'hello',
+  text    : 'hello world!'
+}
+
+var _ 			= require('lodash');
+
+function Post(p) {
```

Comment:
I think we were writing code in case we needed to retrieve posts from the database - perhaps we don't need this function and its corresponding tests?